dbt/models/iasworld/schema.yml

```diff
@@ -3,12 +3,17 @@ version: 2
 
 sources:
   - name: iasworld
+    loaded_at_field: date_parse(wen, '%Y-%m-%d %H:%i:%s.0')
     tables:
       - name: aasysjur
       - name: addn
       - name: addrindx
       - name: aprval
       - name: asmt_all
+        freshness:
+          filter: taxyr = date_format(current_date, '%Y')
+          warn_after: {count: 24, period: hour}
+          error_after: {count: 48, period: hour}
```

Comment:
This config means "warn if the data are 24 hours old, and error if the data are 48 hours old". These are somewhat arbitrary thresholds set based on a quick scan of the data, so I'm definitely open to suggested refinements.

Reply:
**thought:** These timers look good to me! `ASMT_ALL` is the most active table, and we should be getting sqoop extracts every day, so our period of downtime here shouldn't really ever be longer than 48H. I guess we're doing something a little bit unusual in looking at the "updated date" rather than the "loaded date". We currently don't capture the latter, but it might be something worth adding to sqoop in the future.